ant/windows/windows-packager.nsi.in

```diff
@@ -47,8 +47,7 @@ Section
 
   ; Cleanup resources from previous versions
   DetailPrint "Cleaning up resources from previous versions..."
-  RMDir /r "$INSTDIR\demo\js\3rdparty"
-  Delete "$INSTDIR\demo\js\qz-websocket.js"
+  RMDir /r "$INSTDIR"
```

Comment:
Hmm... Let's also use this approach [on Apple](https://github.com/qzind/tray/blob/2.0/ant/apple/apple-postinstall.sh.in#L23:L25) to be consistent.  Linux looks like [it's already doing this](https://github.com/qzind/tray/blob/2.0/ant/linux/linux-installer.sh.in#L95).